config/locales/server.en.yml

```
@@ -1970,6 +1970,7 @@ en:
     edit_post_allowed_groups: "Groups that are allowed to edit posts."
 
     min_trust_to_allow_self_wiki: "The minimum trust level required to make user's own post wiki."
+    self_wiki_allowed_groups: "Groups where users can make their own post wiki."
```

```suggestion
    self_wiki_allowed_groups: "Groups that are allowed to make their own post wiki."
```

Should we follow this pattern? 
edit_post_allowed_groups: "Groups that are allowed to edit posts."

---

I'm not sure it's equivalent to the original, since the suggested change would make "their own" refer to a group's post wiki, but in the original it refers to "user's own post wiki." 🤔